data/verified.ts

```diff
@@ -142,6 +142,8 @@ export const mainnetVerified = [
   '0x38', // cashmere.lens
   '0x0210', // paris.lens
   '0x0160', // ethglobal.lens
+  '0x35', // superproduct.lens
+  '0x6b46', //0xgraham.lens
   ...aaveMembers,
```

Comment:
```suggestion
  '0x0160', // ethglobal.lens
  ...aaveMembers,
```

will be inherited from `aaveMembers`